service/app/controllers/pagemanagement/PageController.java

```diff
@@ -251,4 +255,14 @@
       return Promise.<Result>pure(createCommonExceptionResponse(e, request()));
     }
   }
+
+  private String getQueryString(Map<String, String[]> queryStringMap) {
+    return queryStringMap
+        .entrySet()
+        .stream()
+        .map(p -> p.getKey() + "=" + ((String[]) p.getValue())[0])
```

Comment:
Lets check - if value comes in array - if a=1&b=2&a=3 is given. I expect this to be coming in format : { a=[1,3], b = 2}